Sources/Mocker.swift

```diff
@@ -32,16 +32,19 @@ public struct Mocker {
         case http1_1 = "HTTP/1.1"
         case http2_0 = "HTTP/2.0"
     }
-    
+
+    /// The way Mocker handles unknown URLs
+    public static var mode : Mode = .optout
```

Comment:
```suggestion
    /// The mode defines how unknown URLs are handled. Defaults to `optout` which means requests without a mock will fail.
    public static var mode: Mode = .optout
```

Added a bit more explanation to what the mode does and what the default way is.

Reply:
No problem.